src/status_im2/contexts/quo_preview/profile/profile_card/style.cljs

```diff
@@ -0,0 +1,33 @@
+(ns status-im2.contexts.quo-preview.profile.profile-card.style
+  (:require [quo2.foundations.colors :as colors]))
```

Comment:
move to quo2 please 👍

Reply:
I answered it in https://github.com/status-im/status-mobile/pull/14652#discussion_r1060137675
Please I would do the changes needed, I have no problem with doing changes but you guys should have a definition about this because I did it like than, and I changed it because it was requested, and now i have to change it again.